File: BeaverTinder.Application/Dto/BeaverMatchSearch/LikeRequestDto.cs
using System.ComponentModel.DataAnnotations;

namespace BeaverTinder.Application.Dto.BeaverMatchSearch;

public class LikeRequestDto
{
    [Required]
    [Display(Name = "LikedUserId")]
    public string LikedUserId { get; set; } = default!;
}

File: mobile/lib/dto/likes/like_request_dto.dart
class LikeRequestDto {
  final String likedUserId;

  LikeRequestDto({
    required this.likedUserId
  });

  factory LikeRequestDto.fromJson(dynamic json) =>
      LikeRequestDto(
          likedUserId: json["likedUserId"]
      );
}